contrib/lib/src/ron.rs

```diff
@@ -0,0 +1,179 @@
+use std::ops::{Deref, DerefMut};
+use std::io;
+use std::iter::FromIterator;
+
+use tokio_io::AsyncReadExt;
+
+use rocket::request::Request;
+use rocket::outcome::Outcome::*;
+use rocket::data::{Transform::*, Transformed, Data, FromData, TransformFuture, FromDataFuture};
+use rocket::response::{self, Responder, content};
+use rocket::http::Status;
+
+use serde::{Serialize, Serializer};
+use serde::de::{Deserialize, Deserializer};
+
+
+#[derive(Debug)]
+pub struct Ron<T>(pub T);
+
+impl<T> Ron<T> {
+    #[inline(always)]
+    pub fn into_inner(self) -> T {
+        self.0
+    }
+}
+
+
+//// No idea what the default limit for ron is. Using Json's default of 1MB here.
+const LIMIT: u64 = 1 << 20;
+
+#[derive(Debug)]
+pub enum RonError<'a> {
+    Io(io::Error),
```

Comment:
These should have doc comments (can probably copy almost verbatim from the ones in JSON).